blps/blp-701-http-client-api.md

```diff
@@ -0,0 +1,68 @@
+# Internal HTTP client API
+In order to decouple http-client code from specific implementations,
+we add a façade layer which separate actual http client implementation from our usage.
+The reason we did it, is to not be bound to external libraries, which may not progress fast enough,
+and allow us to replace those libraries at ease, without re-writing large chunks of code.
+
+API is defined and implemented in `cmwell.util.http` package:
+
+```scala
+object SimpleHttpClient {
+
+  // web sockets API is more complex, and out of the scope for this post,
```

Comment:
It's a copy-paste from your post. Are you sure you want to leave this comment?

Reply:
the comment is required. the wording may be changed to "this bloop" rather than "this post". IMHO.

Reply:
in the bloop we can add a short explanation about websockets, but it is too much a "niche" feature in cm-well, that I didn't include an explanation at first.